src/components/controls/toggle-focus.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { connect } from "react-redux";
+import { ImLab } from "react-icons/im";
+import { FaInfoCircle } from "react-icons/fa";
+import { Dispatch } from "@reduxjs/toolkit";
+import Toggle from "./toggle";
+import { SidebarIconContainer, StyledTooltip } from "./styles";
+import { TOGGLE_TREE_FOCUS } from "../../actions/types";
+import { RootState } from "../../store";
+
+
+function ToggleFocus({ tooltip, treeFocus, layout, dispatch, mobileDisplay }: {
+  tooltip: Element;
```

Comment:
I _think_ if you spell this `React.Element`, the type check warning in `controls/controls.tsx` may be silenced? 

```suggestion
  tooltip: React.Element;
```

Reply:
Fixed: https://github.com/nextstrain/auspice/blob/4ad6b43bb1476903308afaabef474cf31cd711e3/src/components/controls/toggle-focus.tsx#L13